package.json

```diff
@@ -41,7 +41,7 @@
     "node-gyp-build": "^4.3.0"
   },
   "optionalDependencies": {
-    "@abandonware/bluetooth-hci-socket": "^0.5.3-8"
+    "@abandonware/bluetooth-hci-socket": "^0.5.3-9"
```

Comment:
please do the bump in other commit

Reply:
@rzr ... last time you commented on an other part and asked to split in two commits ... now this is also wrong ... honestly ... come on ... With my mac it is enormous effort to do these PRs ...

You will get one more PR with anything in single commits and I hope to fullfill your requirements with this.

Please document these requirements somewhere in the Readme(s) that others (and potentielly me in the future) do not run in the same issues again. That all feels like lost time ...